fe/fe-core/src/main/java/com/starrocks/connector/PartitionInfo.java

```diff
@@ -12,9 +12,18 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-
 package com.starrocks.connector;
 
+import com.starrocks.connector.hive.RemoteFileInputFormat;
+
 public interface PartitionInfo {
     long getModifiedTime();
+
+    default RemoteFileInputFormat getFileFormat() {
```

Comment:
why not use RemoteFileInfo?

Reply:
Because PartitionInfo has nothing to do with RemoteFileInfo?  @Youngwb 

PartitionInfo contains
- modification time
- input format of this partition(hms only)
- full path(hms only)